interface/forms/button_ros/report.php

```diff
@@ -0,0 +1,173 @@
+<?php
+//by Art Eaton
+include_once("../../globals.php");
+include_once($GLOBALS["srcdir"]."/api.inc");
+function button_ros_report( $pid, $encounter, $cols, $id) {
+$count = 0;
+$tick =0;
+$obj = formFetch("form_button_ros", $id);
+?>
+
+<span class="title"><center><b>Review of Systems</b></center></span>
+<br></br>
+
+<?php
+ $res1 = sqlStatement("SELECT sex FROM patient_data WHERE pid = $pid");
+$result1 = SqlFetchArray($res1); 
+$gender=$result1['sex'];
+
+
+$patient_reports = "Patient Reports";
```

Comment:
Suggest changing xlt() to...anything else... across the board.  Lots of people accidentally try to use x1t, xlt, xLt, xIt, x|t, and any number of other combinations.  None of which are easy to spot.